content/github/importing-your-projects-to-github/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Importing your projects to GitHub
+intro: You can import your source code to GitHub using a variety of different methods.
```

Comment:
```suggestion
intro: You can import your source code to {% data variables.product.product_name %} using a variety of different methods.
```
Tiny change! I'm switching out the word GitHub for the product variable `{% data variables.product.product_name %}`